Makefile

```diff
@@ -12,6 +12,7 @@ lint: deps-lint
 	--vendor
 
 deps:
+	go get github.com/Masterminds/glide
```

Comment:
This do not belong to this feature. It's a bugfix and should be in master as soon as possible.